easybuild/easyconfigs/p/PROJ/PROJ-4.9.3-foss-2018a.eb

```diff
@@ -0,0 +1,31 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2014-2015 The Cyprus Institute
+# Authors:: Thekla Loizou <t.loizou@cyi.ac.cy>
+# License:: MIT/GPL
+#
+##
+easyblock = 'ConfigureMake'
+
+name = 'PROJ'
+version = '4.9.3'
```

Comment:
@molden Any reason you're not using the most recent `5.0.0`?

As far as I could tell from the release notes, it should be OK to use PROJ 5.0.0 as a drop-in replacement for 4.x (no breaking changes).

I tried this myself as a dependency to `GDAL`, and this worked fine...

Reply:
@molden ping on bumping PROJ to 5.0.0?